pkg/registry/apis/provisioning/jobs/pullrequest/testdata/multiple-files.md

```diff
@@ -0,0 +1,20 @@
+Hey there! 🎉
+Grafana spotted some changes in your dashboard.
+
+
+
+
+See the [original](http://grafana/d/aaa) and [preview](http://grafana/admin/preview) of aaa.json.
```

Comment:
should do this as bullet list or table again?

Reply:
i added a new template that gets used when more than one file